src/observable/dom/AjaxObservable.ts

```diff
@@ -245,6 +245,8 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
   private serializeBody(body: any, contentType: string) {
     if (!body || typeof body === 'string') {
       return body;
+    } else if (typeof FormData !== 'undefined' && body instanceof FormData) {
```

Comment:
I think the check is to ensure FormData is not `undefined`. If it's `undefined`, the instanceof check will fail.